ts/components/MessageComponent.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+
+import { Icon, Left, ListItem, Right, Text } from "native-base";
+import { connectStyle } from "native-base-shoutem-theme";
+import mapPropsToStyleNames from "native-base/src/Utils/mapPropsToStyleNames";
+
+export type OwnProps = {
+  sender: string;
+  subject: string;
+  key: string;
+  date: string;
+};
+
+export type Props = OwnProps;
+
+/**
+ * Implements a component that show a message in the MessagesScreen List
+ */
+class MessageComponent extends React.Component<Props> {
+  public formatDate(date: string): string {
```

Comment:
dates must be formatted based on the locale, we should use a library like moment for this

Reply:
https://github.com/date-fns/date-fns

Reply:
see also https://github.com/catamphetamine/react-time-ago

Reply:
that would be nice to have a method to retrieve the date in its full format (ie. expose a little tooltip on touch)